shaman/models/repos.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+from sqlalchemy import Column, Integer, String, ForeignKey, Boolean, DateTime
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.event import listen
+from sqlalchemy.orm.exc import DetachedInstanceError
+from shaman.models import Base, update_timestamp
+
+
+class Repo(Base):
+
+    __tablename__ = 'repos'
+    id = Column(Integer, primary_key=True)
+    url = Column(String(256))
+    chacra_url = Column(String(256))
+    ref = Column(String(256), index=True)
+    sha1 = Column(String(256), index=True)
+    distro = Column(String(256), nullable=False, index=True)
+    distro_version = Column(String(256), nullable=False, index=True)
+    modified = Column(DateTime, index=True)
+    signed = Column(Boolean(), default=False)
```

Comment:
we might want `signed` because these repos will need to be signed, `type` is not necessary - it is useful only in chacra